clients/ui/frontend/src/app/pages/modelCatalog/screens/ModelDetailsView.tsx

```diff
@@ -43,6 +45,17 @@ const ModelDetailsView: React.FC<ModelDetailsViewProps> = ({
   // Extract all labels from customProperties
   const allLabels = model.customProperties ? getLabels(model.customProperties) : [];
 
+  // Extract validated_on property and split by commas
+  const validatedOnString = model.customProperties
+    ? getCustomPropString(model.customProperties, 'validated_on')
+    : '';
+  const validatedOnPlatforms = validatedOnString
+    ? validatedOnString
+        .split(',')
+        .map((platform) => platform.trim())
+        .filter((platform) => platform.length > 0)
+    : [];
```

Comment:
Should we extract this into a util file and write unit tests for this to cover error cases like consecutive commas or empty values ?